test/unit/gateways/stripe_payment_intents_test.rb

```diff
@@ -373,7 +373,7 @@ def test_purchase_with_google_pay
   def test_purchase_with_shipping_options
     options = {
       currency: 'GBP',
-      customer: @customer,
+      customer: 'abc123',
```

Comment:
Was this included intentionally? seems unrelated

Reply:
@curiousepic there was a failing unit test on Stripe Payment Intents and the reason was that undeclared @customer instance variable, given that was an easy fix I just include that on this PR